portfolio-styling.css

```diff
@@ -0,0 +1,177 @@
+/* Basic stuff for reset, site styles and text sizing */
+
+* {
+	box-sizing: border-box;
+
+}
+
+h1, h2, h3, h4, h5, h6, p, a {
+	margin: 0;
+	padding: 0;
+	text-decoration: none;
+	font-weight: normal;
+}
+
+html {
+	font-size: 15px;
+	font-family: "Courier New";
+}
+
+h1 {
+	white-space: nowrap;
+	display: none;
+}
+
+h2 {
+	font-size: 2.4 rem;
+	width: 100%;
+	text-align: center;
+}
+
+h2.section-title {
+	visibility: hidden;
+	margin: 20px 0 20px 0;
+}
+
+h3 {
+	font-size: 1.8 rem;
+	width: 100%;
+	margin: 5% 0 5% 0;
+	text-align: center;
+}
+
+p {
+	font-size: 1.2 rem;
+	padding: 8%;
+}
+
+a {
+	padding: 2px;
+	white-space: nowrap;
+	font-weight: 500;
+}
+
+a.cheat-sheet-link {
+	text-decoration: underline;
+}
+
+a.portfolio-nav-link {
+	font-size: 1.4rem ;
+}
+
+
+/* Set default backgrounds and text colours  */
+
+body, header, footer {
+	background-color: rgb(0, 0, 0);
+}
+
+h1, h3, p, a {
+	color: rgb(180, 180, 180);
+}
+
+figcaption {
+	text-decoration: none;
+	color: rgb(102, 102, 102);
+}
+
+
+/* Styling for all elements (no custom classes) */
+
+section {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: center;
+}
+
+nav {
+	padding: 10px;
+	border-bottom: 1px solid rgb(180, 180, 180);
+}
+
+article {
+	min-width: 280px;
+	max-width: 400px;
+	margin: 10px;
+	border: 1px solid black;
+	border-radius: 25px;
+	background-color: rgb(51, 51, 51);
+	display: flex;
+	flex-flow: row wrap;
+}
+
+figure {
+	max-width: 80%;
+	margin: 0 auto;
+}
+
+figcaption {
+	text-align: center;
+}
+
+iframe {
+	width: 100%;
+	height: 400px;
+}
+
+footer {
+	margin: 0 auto;
+	padding: 5px 0 10px 0;
+	width: 30%;
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	border-radius: 100px;
+}
+
+
+/* Styling for custom classes ============================== */
+
+section.about-section, article.about-content {
+	justify-content: center;
+}
+
+article.cheat-sheet {
+	max-width: 240px;
+}
+
+figcaption.cheat-sheet-caption {
+	margin: 5% 0 5% 0;
+}
+
+header.portfolio-site-header {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: flex-start;
+	align-items: baseline;
+	position: sticky; /* keep element where it is without removing from document flow */
+	top: 0; /* set a distance from the top to stick to - 0 means always stay at the top */
+}
+
+nav.portfolio-site-nav {
+	display: flex;
+	flex-flow: row wrap;
+	justify-content: space-evenly;
+	flex-grow: 2;
+}
+
+img.about-image, img.project-image {
+	width: 100%;
+	max-width: 300px;
+	margin: 0 auto;
+	border-radius: 25px;
+}
+
+img.logo {
+	width: 50px;
+	height: 50px;
+}
```

Comment:
the icons could do with some spacing between each other on mobile at least, on the smallest width it looks like this:
![image](https://user-images.githubusercontent.com/25844355/125764017-a2bebc7f-4b3b-4cab-a282-4d5ec79b9eb8.png)